Formula/rosa-cli.rb

```diff
@@ -0,0 +1,31 @@
+class RosaCli < Formula
+  desc "RedHat OpenShift Service on AWS (ROSA) command-line interface"
+  homepage "https://www.openshift.com/products/amazon-openshift"
+  url "https://github.com/openshift/rosa.git",
```

Comment:
Do we need a git checkout? Can we use a tarball?

Reply:
Yes, we can.